src/prog_algs/metrics/uncertain_data_metrics.py

```diff
@@ -9,7 +9,7 @@
 from warnings import warn
 from ..uncertain_data import UncertainData, UnweightedSamples
 
-def calc_metrics(data, ground_truth = None, **kwargs):
+def calc_metrics(data : array, ground_truth : float = None, **kwargs) -> dict:
```

Comment:
```suggestion
def calc_metrics(data : UncertainData, ground_truth : float = None, **kwargs) -> dict:
```

I think we want uncertain data to be the hint here